searchcore/src/vespa/searchcore/fdispatch/search/fnet_search.cpp

```diff
@@ -886,7 +886,7 @@ FastS_FNET_Search::CheckCoverage()
             cntNone++;
         }
     }
-    if ((cntNone > 0) && (cntNone != _nodes.size())) {
+    if ((cntNone >= _dataset->getSearchableCopies()) && (cntNone != _nodes.size())) {
         activeDocs += cntNone * activeDocs/(_nodes.size() - cntNone);
```

Comment:
We must adjust cntNode with (searchableCopies - 1)

Reply:
Fixed